app/controllers/device_tokens_controller.rb

```diff
@@ -0,0 +1,19 @@
+class DeviceTokensController < PostsController
```

Comment:
Why is this controller inheriting from `PostsController`?

Reply:
good catch

Reply:
waaaaaaat 👓